rulebook-core/src/main/java/com/deliveredtechnologies/rulebook/model/runner/AbstractRuleBookRunner.java

```diff
@@ -51,20 +51,20 @@ public void run(NameValueReferableMap facts) {
       RuleBook ruleBook = _prototypeClass.newInstance();
       List<Class<?>> classes = getPojoRules();
       for (Class<?> rule : classes) {
-        try {
-          getAnnotatedField(com.deliveredtechnologies.rulebook.annotation.Result.class, rule).ifPresent(field ->
-              ruleBook.setDefaultResult(_result.getValue())
-          );
-          String name = getAnnotation(com.deliveredtechnologies.rulebook.annotation.Rule.class, rule).name();
-          if (name.equals("None")) {
-            name = rule.getSimpleName();
-          }
-          Rule ruleInstance = new AuditableRule(new RuleAdapter(rule.newInstance()), name);
-          ruleBook.addRule(ruleInstance);
-          ((Auditable)ruleInstance).setAuditor(this);
-        } catch (IllegalAccessException | InstantiationException ex) {
-          LOGGER.warn("Unable to create instance of rule using '" + rule + "'", ex);
+        getAnnotatedField(com.deliveredtechnologies.rulebook.annotation.Result.class, rule).ifPresent(field ->
+            ruleBook.setDefaultResult(_result.getValue())
+        );
+        String name = getAnnotation(com.deliveredtechnologies.rulebook.annotation.Rule.class, rule).name();
+        if (name.equals("None")) {
+          name = rule.getSimpleName();
         }
+        Object ruleInstance = getRuleInstance(rule);
+        if (ruleInstance == null) {
+          return;
```

Comment:
If a rule could not be loaded, then it should be omitted. I this also might be the reason why some of the unit tests are failing with this PR.

Reply:
I'm surprised it changed existing behavior since the existing AbstractRuleBookRunner code logs an error and the method ends (since the catch block is at the end and the method has nothing after it). The only difference here is that the error log happens inside the new getRuleInstance() method, so functionally nothing should have changed. I'll run unit tests to see what's going on though.

Reply:
It's in a loop. The return breaks the loop. Previously, the exception would be caught and the loop would continue.